common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -6,12 +6,18 @@ GeoFencing
 
 ArduPilot supports failsafes generated by several types of GeoFences (boundaries described by latitude/longitude and/or altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. Types of GeoFences supported varies by vehicle.
 
+[site wiki="copter,rover"]
+For Copter, the Cylindrical Geo-Fence is a simple "tin-can" centered around HOME. In addition, there can also be inclusion and exclusion zones drawn, either circular or polygonal, or both, at various locations. 
+
+Inclusion and Exclusion Zones are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the  FENCES item in the drop down box, just like Missions or Rally Points are planned.
+[/site]
+
 .. toctree::
     :maxdepth: 1
 
 [site wiki="copter,rover"]
     Cylindrical Geo-Fencing Failsafe <common-ac2_simple_geofence>
-    Polygon Geo-Fencing Failsafe <common-polygon_fence>
+    Inclusion/Exclusion Zone Failsafe <common-polygon_fence>
```

Comment:
I think we should stick to fewer words if possible.  I prefer "Fence" to "Zone".  I also prefer "Fence" to "Geo-Fence". 

I sometimes use the word "Stay-in" and "Stay-out" although I'm not sure we use these on the wiki.  I don't think we should anymore because "inclusion/exclusion" is what MP (and perhaps other GCSs) use to to describe these fences.

So.. I think maybe we've got the "Cylindrical fence" and "Inclusion/Exclusion Circular fences" and "Inclusion/Exclusion polygon fences".

Reply:
Changed for copter/rover.....left plane as GeoFence......
MP is not aligned with this....still uses GeoFence on the Config/GeoFence screen...